_posts/2016-07-28-community-testing-day-on-20160805.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Community Testing Day on August 5th
+layout: post
+author: Chongrui Zhang
+---
+
+The Pulp 2.10 GA is [scheduled for Aug 16, 2016](https://pulp.plan.io/projects/pulp/wiki/Release_Schedule). To deliver a robust Pulp release, the Pulp QE and Pulp team have organized a Community Test day where the Pulp community can come together to test Pulp. This post further details the event and how to participate it.
+
+# What is a Community Testing Day and why host it?
+
+According to the Fedora blog, organizing a test day "is a great way to help expose your project and bring more testers to trialing a new update before it goes live".
+
+The main objective of hosting this Pulp Test Day is to identify as many issues as possible early on with the help of the community, get them fixed as quickly as possible and thus have a more stable release.
+
+# What to test?
+
+The focus of the test day will be the new features included in Pulp 2.10. The issues fixed in 2.10 can also be verified that day.  A list of these features will be provided, and participants can sign up for the features they want to test. The attendees are welcomed to test existing features as well. Members can also participate in the test day by adding tests to our automation test suite (Pulp Smash) or by submitting fixes to the reported bugs. All issues and bugs will be filed and triaged to prepare for 2.10 GA.
+
+# When to host?
+
+The Pulp QE team has tentatively decided to host the day on **Aug 05, 9am-5pm EDT**.
+
+# Where to host?
+
+The major communication approach will be through IRC. Participants are encouraged to join **channels #pulp and #pulp-dev on freenode**. The Pulp email list pulp-list@redhat.com can also be used for communications. In order to expand the audience of this Test Day, members of both the open source community as well as downstream users will be notified via email.
+
+# Getting Ready
+
+Here are some instructions for you to install or update your Pulp.
+
+- Install fresh Pulp 2.10 beta according to the official documentation: <http://docs.pulpproject.org/user-guide/installation/index.html>
+
+- Optionally, you can use Ansible to install Pulp onto Fedora 23 by the following commands:
+
+  ```
+        #!/usr/bin/env sh
+        set -euo pipefail
+        dnf update -y
+        dnf install -y git vim ansible python-dnf libselinux-python
+        mkdir code
+        cd code
+        git clone https://github.com/pulp/pulp_packaging
+        cd pulp_packaging
+        echo localhost > hosts
+        ansible-playbook -i hosts --connection local -e pulp_version='2.9' \
+            ci/ansible/pulp_server.yaml
+  ```
+
+- You can also update your current Pulp to Pulp 2.10 beta. Here is a link about how to upgrade to 2.8. You can follow the instructions to upgrade to 2.10: <http://docs.pulpproject.org/user-guide/release-notes/2.8.x.html#upgrade-instructions>.
```

Comment:
s/current/current, non-production/
It's really important that users understand 2 things. 1) that they should not use a production instance for beta/rc testing. 2) That we don't support upgrading out of a beta or RC into another beta/RC/GA version. I think a sentence or paragraph should be added conveying these things because it's important.